qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/UserSteps.java

```diff
@@ -0,0 +1,159 @@
+package org.openmrs.contrib.qaframework.automation;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Random;
+import java.util.Map.Entry;
+
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+import org.openmrs.contrib.qaframework.RunTest;
+import org.openmrs.contrib.qaframework.page.AddEditUserPage;
+import org.openmrs.contrib.qaframework.page.HomePage;
+import org.openmrs.contrib.qaframework.page.ManageUserPage;
+
+public class UserSteps  extends Steps {
```

Comment:
I  would not have changed this method to userStep . the former name had a more meaning.

Reply:
>  I would not have changed this method to userStep . the former name had a more meaning

I am fine with this! cc: @mherman22